Kickstarter-iOS/ViewModels/AppDelegateViewModel.swift

```diff
@@ -313,11 +273,11 @@ AppDelegateViewModelOutputs {
     self.pushTokenSuccessfullyRegistered = self.deviceTokenDataProperty.signal
       .map(deviceToken(fromData:))
       .ksr_debounce(.seconds(5), on: AppEnvironment.current.scheduler)
-      .switchMap {
-        AppEnvironment.current.apiService.register(pushToken: $0)
+      .switchMap { token in
+        AppEnvironment.current.apiService.register(pushToken: token)
           .demoteErrors()
+          .map { _ in token }
```

Comment:
Returning the token here for easy debugging in the console.

Reply:
Can we return this token even if the registration API request fails? (It tends to fail all the time when running on a debug build)/